lib/condor.py

```diff
@@ -34,13 +34,13 @@
 
 
 # pylint: disable-next=no-member
-def get_schedd(vargs: Dict[str, Any]) -> classad.ClassAd:
+def get_schedd_list(vargs: Dict[str, Any]) -> List[classad.ClassAd]:
     """get jobsub* schedd names from collector, pick one."""
```

Comment:
Could you please update this docstring (and add ones to the two new functions `get_schedd_names` and `get_schedd`) so that in the future, it's very clear which one of these to use?

For the future, do we think this is expansive enough to warrant a `Class` of its own?